config/github.yml

```diff
@@ -1,36 +1,31 @@
 # listener services that you can opt into for a repository
 services:
+    _defaults:
+        autowire: true      # Automatically injects dependencies in your services.
+
     app.subscriber.status_change_by_comment_subscriber:
-        class: AppBundle\Subscriber\StatusChangeByCommentSubscriber
-        arguments:
-            - '@app.status_api'
-            - '@logger'
+        class: App\Subscriber\StatusChangeByCommentSubscriber
+        public: true
 
     app.subscriber.status_change_on_push_subscriber:
-        class: AppBundle\Subscriber\StatusChangeOnPushSubscriber
-        arguments:
-            - '@app.status_api'
+        class: App\Subscriber\StatusChangeOnPushSubscriber
+        public: true
 
     app.subscriber.needs_review_new_pr_subscriber:
-        class: AppBundle\Subscriber\NeedsReviewNewPRSubscriber
-        arguments:
-            - '@app.status_api'
+        class: App\Subscriber\NeedsReviewNewPRSubscriber
+        public: true
 
     app.subscriber.bug_label_new_issue_subscriber:
-        class: AppBundle\Subscriber\BugLabelNewIssueSubscriber
-        arguments:
-            - '@app.status_api'
+        class: App\Subscriber\BugLabelNewIssueSubscriber
+        public: true
 
     app.subscriber.auto_label_pr_from_content_subscriber:
-        class: AppBundle\Subscriber\AutoLabelPRFromContentSubscriber
-        arguments:
-            - '@app.github.cached_labels_api'
+        class: App\Subscriber\AutoLabelPRFromContentSubscriber
+        public: true
 
     app.subscriber.status_change_by_review_subscriber:
-        class: AppBundle\Subscriber\StatusChangeByReviewSubscriber
-        arguments:
-            - '@app.status_api'
-            - '@logger'
+        class: App\Subscriber\StatusChangeByReviewSubscriber
+        public: true
```

Comment:
Could we remove this file entirely? And fully register everything as a class name (i.e.remove the snake case ids). I'd rather fully modernize the code base (and services) now, test it, and be done :)

Reply:
Later